README.md

```diff
@@ -36,3 +35,6 @@ The Developers' Guide is continuously updated and there are several parts still
 * Text about JBS, triage, priorities, status
 * List JTReg `@key` conventions for different areas
 * Document best practices around TEST.properties usage. See [PR#30](https://github.com/openjdk/guide/pull/30#issuecomment-714589551)
+* How to work with the code in an IDE
```

Comment:
This would be a great addition. There's already the [IDE Tooling & Support Group](https://openjdk.java.net/groups/ide-support/), but AFAICT it hasn't produced any documentation yet (though there's a [post](https://mail.openjdk.java.net/pipermail/ide-support-dev/2020-April/000030.html) by Bradford Wetmore on his NetBeans setup).

Reply:
Yes, I have Brad's mail in my inbox and it's on my todo list to turn that into a section in the Guide.